From: Emmanuel Arias <emmanuelarias30@gmail.com>
Date: Wed, 11 Dec 2019 13:51:16 -0300
Subject: Use Debian provided /etc/ssl/certs/ca-certificates.cr

---
 certifi/core.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/certifi/core.py b/certifi/core.py
index 7271acf..53404e1 100644
--- a/certifi/core.py
+++ b/certifi/core.py
@@ -12,4 +12,4 @@ import os
 def where():
     f = os.path.dirname(__file__)
 
-    return os.path.join(f, 'cacert.pem')
+    return '/etc/ssl/certs/ca-certificates.crt'
